pages/survey.py

```diff
@@ -107,16 +105,27 @@ def update_user_data(page_finished = "", column_idx = -1):
 def count_words(text):
     return len(text.split())
 
-def record_data_clear_state(keys_list = [], header=False, survey_type = ""):
+def record_data_clear_state(keys_list = [], survey_page_num = -1):
     # convert the data from dict to tuple
-    responses = []
-    keys = keys_list
-    for key in keys:
+    responses = {}
+    for key in keys_list:
         if key in st.session_state:
-            responses.append((key, st.session_state[key]))
-    logger.write_survey_response(responses, header, survey_type)
+            responses[key] = st.session_state[key]
+    
+    survey_worksheet = st.session_state['sheet']
+
+    if survey_page_num == 1:
+        survey_worksheet = exponential_backoff(st.session_state['sheet'].worksheet, "Survey Page 1")
+    elif survey_page_num == 2:
```

Comment:
can we create a variable e.g. `survey_page_id` for the survey page name and make it more readable instead of survey page 1 2 and 3? and also reuse that variable as the argument instead of hard coding it as "Survey Page 1"?